hw/ip_templates/racl_ctrl/dv/sva/racl_ctrl_bind.sv.tpl

```diff
@@ -0,0 +1,23 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+module racl_ctrl_bind;
+
+  bind ${module_instance_name} tlul_assert #(
+    .EndpointType("Device")
+  ) tlul_assert_device (
+    .clk_i,
+    .rst_ni,
+    .h2d  (tl_i),
+    .d2h  (tl_o)
+  );
+
+  bind ${module_instance_name} racl_ctrl_csr_assert_fpv racl_ctrl_csr_assert (
```

Comment:
Do the others also need to be templated?

Reply:
Hmm, good point. I think the name ending in "_fpv" ends up being based on the `mod_name` defined at line 42 of `gen_fpv.py` and that, in turn, depends on the block that gets passed in from `regtool.py` (line 273). *That* name will be extracted from the expanded block.

I think!

So I'll switch that over, but don't think we need to care about the instance name. Let me know if I've got anything backwards :-)